projects/microphysics/scripts/prognostic_run.py

```diff
@@ -40,6 +42,9 @@ def get_env(args):
     default="",
     help="A unique tag. Can be used to look-up these outputs in subsequent timesteps.",
 )
+parser.add_argument("--save-zarr", action="store_true", default=False)
```

Comment:
You've modified the default behavior from storing zarrs to not storing zarrs, is that intended?

Reply:
It was...we should be fine. Better not to save loads of redundant data by default,